### ISSUE TITLE
Missing texture assets

### ISSUE THREAD

**Issue by PhyDoesStuff** [l33tmaan/vs-expandedfoods#20]
Mod version: v1.5.3-pre.11
Game version: 1.16.5

found in client-main.txt

```
10.6.2022 19:46:16 [Warning] Texture asset 'expandedfoods:textures/item/food/mushroom/partbaked/fieldmushroom.png' not found (defined in Item expandedfoods:cookedchoppedmushroom-fieldmushroom-partbaked).
10.6.2022 19:46:16 [Warning] Texture asset 'expandedfoods:textures/item/food/mushroom/partbaked/flyagaric.png' not found (defined in Item expandedfoods:cookedchoppedmushroom-flyagaric-partbaked).
10.6.2022 19:46:16 [Warning] Texture asset 'expandedfoods:textures/food/mushroom/partbaked/reishi.png' not found (defined in Shape file expandedfoods:item/food/mushroom/chopped/cooked/reishi).
10.6.2022 19:46:16 [Warning] Texture asset 'expandedfoods:textures/food/mushroom/partbaked/shiitake.png' not found (defined in Shape file expandedfoods:item/food/mushroom/chopped/cooked/shiitake).
10.6.2022 19:46:16 [Warning] Texture asset 'expandedfoods:textures/item/food/mushroom/perfect/fieldmushroom.png' not found (defined in Item expandedfoods:cookedchoppedmushroom-fieldmushroom-perfect).
10.6.2022 19:46:16 [Warning] Texture asset 'expandedfoods:textures/item/food/mushroom/perfect/flyagaric.png' not found (defined in Item expandedfoods:cookedchoppedmushroom-flyagaric-perfect).
10.6.2022 19:46:16 [Warning] Texture asset 'expandedfoods:textures/item/food/mushroom/charred/fieldmushroom.png' not found (defined in Item expandedfoods:cookedchoppedmushroom-fieldmushroom-charred).
10.6.2022 19:46:16 [Warning] Texture asset 'expandedfoods:textures/item/food/mushroom/charred/flyagaric.png' not found (defined in Item expandedfoods:cookedchoppedmushroom-flyagaric-charred).
10.6.2022 19:46:16 [Warning] Texture asset 'wildcraft:textures/item/liquid/potentwine.png' not found (defined in Item expandedfoods:wildpotentspiritportion-huckleberry).
10.6.2022 19:46:16 [Warning] Texture asset 'wildcraft:textures/item/liquid/potentwine.png' not found (defined in Item expandedfoods:wildpotentspiritportion-snowberry).
10.6.2022 19:46:16 [Warning] Texture asset 'wildcraft:textures/item/liquid/potentwine.png' not found (defined in Item expandedfoods:wildpotentspiritportion-brambleberry).
10.6.2022 19:46:16 [Warning] Texture asset 'wildcraft:textures/item/liquid/potentwine.png' not found (defined in Item expandedfoods:wildpotentspiritportion-cloudberry).
10.6.2022 19:46:16 [Warning] Texture asset 'wildcraft:textures/item/liquid/potentwine.png' not found (defined in Item expandedfoods:wildpotentspiritportion-dogrose).
10.6.2022 19:46:16 [Warning] Texture asset 'wildcraft:textures/item/liquid/potentwine.png' not found (defined in Item expandedfoods:wildpotentspiritportion-elderberry).
10.6.2022 19:46:16 [Warning] Texture asset 'wildcraft:textures/item/liquid/potentwine.png' not found (defined in Item expandedfoods:wildpotentspiritportion-gooseberry).
10.6.2022 19:46:16 [Warning] Texture asset 'wildcraft:textures/item/liquid/potentwine.png' not found (defined in Item expandedfoods:wildpotentspiritportion-honeysuckle).
10.6.2022 19:46:16 [Warning] Texture asset 'wildcraft:textures/item/liquid/potentwine.png' not found (defined in Item expandedfoods:wildpotentspiritportion-raspberry).
10.6.2022 19:46:16 [Warning] Texture asset 'wildcraft:textures/item/liquid/potentwine.png' not found (defined in Item expandedfoods:wildpotentspiritportion-strawberry).
```

**Comment by l33tmaan** [comment]
This should be fixed in 1.5.3 - check it out and let me know if you still see these warnings.

**Comment by PhyDoesStuff** [comment]
Confirmed, no longer seeing these entries in 1.5.3.
